readthedocs/projects/models.py

```diff
@@ -306,6 +306,11 @@ class Project(models.Model):
         default=False,
         help_text='If checked, do not show advertising for this project',
     )
+    is_spam = models.NullBooleanField(
+        _('Is spam?'),
+        default=None,
+        help_text=_('Manually marked as (not) spam'),
+    )
```

Comment:
This is a confusing name, and implies that projects that _are_ spam will have this `True`. We want a better name here, perhaps `disable_spam_checks` or something?

Reply:
I'm 👍  on merging this once we fix the name though. Happy to see it go out next release.

Reply:
He he, I did it this way on purpose. The idea is that we can manually mark a project as spam (`True`) or as non-spam (`False`). The default value is `None` because the project wasn't manually verified and we don't know (yet) if it's spam or not.

Reply:
I just added the second part of this work that reflects better the `is_spam=True` use case. See [readthedocs/readthedocs-ext@`abd414c` (#483)](https://github.com/readthedocs/readthedocs-ext/pull/483/commits/abd414c61959b6f4de35874c00caa432e458c1bc)

Reply:
Gotcha, makes sense then. 👍